### PR TITLE
Fix: xlsx export format for financial reports

### PR DESCRIPTION
**Pull request by rvpasari**
Changed

- export format for excel files in data export; added header items and bolded header row, table borders

https://app.asana.com/0/1192407975826276/1199966341817852

### CODE REVIEW

**Comment by rvpasari** [comment]
Depends on: 

https://github.com/Bloomstack/erpnext/pull/1079

File: frappe/desk/query_report.py [hunk 1]
```diff
@@ -327,17 +327,24 @@ def export_query():
...
-def build_xlsx_data(columns, data, visible_idx, include_indentation):
+def build_xlsx_data(columns, data, visible_idx, include_indentation, filters=None, report_name=None):
 	result = [[]]
 
+	# add headers from the filters as list items
+	headers = [["Report: {0}".format(report_name)]]
+	date_range = filters.get('from_date', "") + " - " + filters.get('to_date',"")
+	company = filters.get('company', None)
+	headers.append(["Company: {0}".format(company)])
+	headers.append(["Period: {0}".format(date_range)])
```
Comment (hrwX) [inline frappe/desk/query_report.py]:
```suggestion
	headers = [
		["Report: {0}".format(report_name)],
		["Company: {0}".format(filters.get('company', None))],
		["Period: {0} - {1}".format(filters.get("from_date", ""), filters.get("to_date",""))]
	]
```

File: frappe/utils/xlsxutils.py [hunk 2]
```diff
@@ -39,6 +42,28 @@ def make_xlsx(data, sheet_name, wb=None):
...
+		no_of_values = sum([1 if cell.value!=None else 0 for cell in row])
+		for cell in row:
+
+			#change header to bold if there is only 1 value in the whole row - usually header items
+			if no_of_values == 1:
+				cell.font = Font(bold=True)
+
+			#change header row of table to bold
+			if no_of_values > 1 and header_flag:
+				cell.font = Font(bold=True)
```
Comment (hrwX) [inline frappe/utils/xlsxutils.py]:
```suggestion
			#change header row of table to bold
			if (no_of_values == 1) or (no_of_values > 1 and header_flag):
				cell.font = Font(bold=True)
```

**Review by hrwX — DISMISSED** [review]

File: frappe/desk/query_report.py [hunk 3]
```diff
@@ -327,17 +327,24 @@ def export_query():
...
-def build_xlsx_data(columns, data, visible_idx, include_indentation):
+def build_xlsx_data(columns, data, visible_idx, include_indentation, filters=None, report_name=None):
 	result = [[]]
 
+	# add headers from the filters as list items
+	headers = [
+		["Report: {0}".format(report_name)],
+		["Company: {0}".format(filters.get('company', None))],
+		["Period: {0} - {1}".format(filters.get('from_date', ""), filters.get('to_date',""))]
+		]
```
Comment (hrwX) [inline frappe/desk/query_report.py]:
```suggestion
	]
```

**Review by hrwX — CHANGES_REQUESTED** [review]

**Review by hrwX — APPROVED** [review]

**Review by sahil28297 — APPROVED** [review]

**Pull request merged**